rules.js

```diff
@@ -8,11 +8,9 @@ module.exports = {
     'color-no-invalid-hex': true,
     'font-family-name-quotes': 'always-unless-keyword',
     'font-weight-notation': 'numeric',
-    'media-feature-no-missing-punctuation': true,
-    'declaration-block-no-ignored-properties': true,
     'number-leading-zero': 'never',
-    'selector-no-empty': true,
     'string-quotes': 'single',
+    'rule-empty-line-before': ['always', { 'except': 'first-nested'} ],
```

Comment:
Why don't we rely on `stylelint-config-standard` for this, just as before? We're changing the behavior here, and tests are failing (we're forcing space after comments and changing 'always-multi-line' to 'always')